packages/components/src/DateTimePickers/pickers/TimePicker/TimePicker.scss

```diff
@@ -0,0 +1,26 @@
+.container {
+    padding: $padding-smaller 0;
+    background-color: $white;
+
+    button {
+        margin: 1px;
```

Comment:
add margin 1px so there's space to show focus outline

Reply:
You can remove the outline as its replaced by the background. And remove the margin.